pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant

```diff
@@ -130,8 +130,6 @@ FROM system.span_configurations
 WHERE end_key > (SELECT crdb_internal.table_span($t_id)[1])
 ORDER BY start_key
 ----
-/Table/110  {"gcPolicy": {"ttlSeconds": 90001}, "numReplicas": 3, "rangeMaxBytes": "67108864", "rangeMinBytes": "1048576"}
```

Comment:
The removal of these two lines seems suspicious - why did it happen?

Reply:
Thanks for catching this! `--rewrite` logic tests sometimes removes unrelated parts of test data and I didn't notice this one. It will brought back within this PR #111920